.github/workflows/docker.yml

```diff
@@ -0,0 +1,74 @@
+name: Create and publish a Docker images
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
if i create RC tag or other, eg `v4.0.0-beta.2`, will the flow work properly, or that tag syntax will fail on one of further steps ? or maybe even override and create `4.0.0` ?